cmd/tempo-query/tempo/plugin.go

```diff
@@ -379,7 +399,14 @@ func createTagsQueryParam(service string, operation string, tags map[string]stri
 }
 
 func (b *Backend) lookupTagValues(ctx context.Context, span opentracing.Span, tagName string) ([]string, error) {
-	url := fmt.Sprintf("%s://%s/api/search/tag/%s/values", b.apiSchema(), b.tempoBackend, tagName)
+	var url string
+
+	if b.QueryServicesDuration == nil {
+		url = fmt.Sprintf("%s://%s/api/search/tag/%s/values", b.apiSchema(), b.tempoBackend, tagName)
+	} else {
+		startTime, endTime := b.calculateTimeRange()
+		url = fmt.Sprintf("%s://%s/api/search/tag/%s/values?start=%d&end=%d", b.apiSchema(), b.tempoBackend, tagName, startTime, endTime)
```

Comment:
should this be controlled with a config value? does jaeger not pass similar params when it searches its own backend?

Reply:
It pass the parameters only when you search for traces, but for do an initial search we need the service name.